Include serialize feature

feikesteenbergen opened the pull request:
For inspecting statements and storing them (in PostgreSQL as json for example) it is sometimes useful to serialize the parsed query.

This feature flag allows this to happen.

feikesteenbergen commented:
A bit of a discussion item perhaps, but I for one would really appreciate this in this library.

seanlinsley commented:
This seems reasonable, though I wonder if it needs to be a feature or should be unconditionally enabled.

I wonder why CI is failing; the error seems to be unrelated to this change.

feikesteenbergen commented:
@seanlinsley pinging you as I rebased the PR on 6.1.0 and removed the CI failure.

I still wonder if this should be behind a feature flag or not, especially as no additional dependencies are required.

seanlinsley commented:
I think it should be fine to enable it by default.

Note that when making that update, the generated protobuf definitions are now stored in the repo as `src/protobuf.rs`. A `cargo build` should update it automatically. If it doesn't, that probably means you need to `apt-get install protobuf-compiler` or similar

seanlinsley commented:
Looks good, sorry for the delay reviewing.
